test/CMakeLists.txt

```
@@ -1283,6 +1285,17 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_gcas
                           -r 2023080215 2023080321"
                           ioda_GCAS_NO2_20230802.nc ${IODA_CONV_COMP_TOL})
 
+ecbuild_add_test( TARGET  test_${PROJECT_NAME}_pandora
+                  TYPE    SCRIPT
+                  ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
+                  COMMAND bash
+                  ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                          netcdf
+                          "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/pandora_2ioda.py
+                          -i testinput/Pandora57s1_BoulderCO_L2_rnvs3p1-8.txt
+                          -o testrun/Pandora57s1_BoulderCO_L2_rnvs3p1-8.nc
+                          -r 2023080115 2023080121"
```

```suggestion
                          --time_range 2023080115 2023080121"
```

Personal preference, but I usually try to minimize the use of the shorthand options as they can be confusing, but feel `-i` and `-o` are always good to leave shorthand